src/cmd/extrinsics/decode.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018-2020 Parity Technologies (UK) Ltd.
```

Comment:
We might need to move things around a bit, since `decode` is not an extrinsic so does not belong in the `extrinsics` folder. However since it does require the `transcode` module, perhaps we could lift this command and that module up to the root.